curvefs/src/client/s3/disk_cache_manager.cpp

```diff
@@ -450,8 +450,11 @@ void DiskCacheManager::InitMetrics(const std::string &fsName) {
     cacheWrite_->InitMetrics(metric_);
     cacheRead_->InitMetrics(metric_);
     // this function move to here from init，
-    // Otherwise, you can't get the original metric
-    SetDiskInitUsedBytes();
+    // Otherwise, you can't get the original metric.
+    // SetDiskInitUsedBytes may takes a long time,
+    // so use a separate thread to do this.
+    Thread t(&DiskCacheManager::SetDiskInitUsedBytes, this);
+    t.detach();
```

Comment:
It is not a good idea to use a detach thread，if thead exec function SetDiskInitUsedBytes after DiskCacheManager destruct，it will cause an exception.

Reply:
+1